person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
```

Comment:
There should be an individual .rb file for each class (Person, Student, and Teacher 🐐 Well done! 

Kindly rename your teachers.rb to teacher.rb so that the naming of the class is done properly 🔥

Reply:
Person class is as expected 📝 Great work!